### ISSUE TITLE
'-' in layer name produces crash

### ISSUE THREAD

**Issue by iperov** [plaidml/plaidml#241]
>Conv2D (10, kernel_size=(3,3), strides=(1,1), padding='valid', name= **"conv4-2"**)

```
plaidml.exceptions.Unknown: syntax error, unexpected -, expecting "," or ) : fun
ction (
  X_I_0[X_I_0_0, X_I_0_1, X_I_0_2, X_I_0_3],
  X_I_1[X_I_1_0, X_I_1_1, X_I_1_2, X_I_1_3],
  X_I_2[X_I_2_0]
) -> (
  X_T6
) {
  [[pid(conv4-2)]] X_T4[n, x0, x1, co : 1, 62, 62, 10] = +(X_I_0[n, k0 + x0, k1
+ x1, ci] * X_I_1[k0, k1, ci, co]);
  [[pid(Reshape)]] X_T5 = reshape(X_I_2, X_I_0_0, X_I_0_0, X_I_0_0, X_I_2_0);
  [[pid(Add)]] X_T6 = add(X_T4, X_T5);
}
```

p.s. thank you for plaidML !!!! 
I am using plaidML as backend of keras.
Please keep to improve it.

**Comment by brianretford** [comment]
Hey Mars, can you fix this annotation parsing?

Iperov - thanks! We are definitely going to continue working on it - hope to have more to announce soon!

**Comment by iperov** [comment]
@brianretford now [DeepFaceLab](https://github.com/iperov/DeepFaceLab) with plaidML backend works on any card, even IntelHD graphics :D

![works_on_intelhd](https://user-images.githubusercontent.com/8076202/53021307-5b0cea80-3472-11e9-97e6-e5e2a4ae8b4f.jpg)

again thank you very much,

**Comment by mars1026** [comment]
We've got a fix for this problem internally - it should be part of the next release. Thanks for bringing it up.

**Comment by brianretford** [comment]
Apparently the fix for this got lost, Mars will reverify.
